pkg/COPYRIGHT

```diff
@@ -0,0 +1,28 @@
+The files in this directory and the "pkg/fedora" directory are distributed
```

Comment:
This paragraph is technically unnecessary, since it's already covered by the top-level `COPYING` file.  But I figured that when it came to copyright and licensing, it was better to be overly-clear.